lesscss/improved-initiative.less

```diff
@@ -707,6 +707,7 @@ body>.login.button {
     }
     hr {
         margin: @small-spacer;
+        margin-bottom: 25px;
```

Comment:
I don't think you want extra margin on the `<hr>` elements, this will add a lot of extra whitespace between statblock sections. Another approach could be to add padding at the bottom of the right column. TBH, the 'Login' button gets in the way slightly on purpose to encourage users to log in. It's technically a [dark pattern](https://darkpatterns.org/), so I could be persuaded to change it, but I think a better layout will take a bit more work.

Reply:
Somehow it didn't occur to me this would space out the other `<hr>`'s as well. Adding padding to the bottom of the right column pads it in such a way that it reduces the overall viewport. Adding the padding to c-statblock seems like the best option, though it does pad the previews as well, which isn't great.

Personally, I don't think there's much value to letting the button obscure the text. My workaround for one of my monsters was to just add a bunch of whitespace to its last action, so as a dark pattern (if that's really what you want) it's pretty weak.

Reply:
Yeah, I don't like that this will affect all statblocks (including the left-column statblock and preview statblocks), and it will also create unnecessary padding when the login button is hidden. I think you should revert this change for this PR, the rest of it looks good though.

Reply:
I understand, reverted.